en/docs/reference/connectors/exporting-artifacts.md

```diff
@@ -7,17 +7,17 @@
 To bundle a Connector into a CApp, a `Connector Exporter Project` is required. 
 
 1. Navigate to **File** -> **New** -> **Other** -> **WSO2** -> **Extensions** -> **Project Types** -> **Connector Exporter Project**.<br> 
-  <img src="{{base_path}}/assets/img/integrate/connectors/connector-exporter-project-1.jpg" title="Add Connector Exporter Project" width="400" alt="Add Connector Exporter Project" />
+   <img src="../../../../assets/img/integrate/connectors/connector-exporter-project-1.jpg" title="Add Connector Exporter Project" width="400" alt="Add Connector Exporter Project" />
 
 2. Enter a name for the Connector Exporter Project. 
 
 3. In the next screen select, **Specify the parent from workspace** and select the specific Integration Project you created from the dropdown. 
-  <img src="{{base_path}}/assets/img/integrate/connectors/connector-exporter-project-naming.png" title="Naming Connector Exporter Project" width="400" alt="Naming Connector Exporter Project" />
+   <img src="../../../../assets/img/integrate/connectors/connector-exporter-project-naming.png" title="Naming Connector Exporter Project" width="400" alt="Naming Connector Exporter Project" />
```

Comment:
@samgnaniah , why have you removed base_path?

Reply:
base_path doesn't work for include pages hence the change here. These were all broken due to base_path used here - maybe we have changed it accidentally at some point.